sites/footers/footers-en.html

```diff
@@ -46,6 +46,7 @@ <h2>Version 4.0</h2>
 	<li>Removed "Top of page" anchor.</li>
 	<li>Complete rework of the footer's headings for accessibility.</li>
 	<li>GCWeb Jekyll specific: Footer skip link should always remain.</li>
+	<li><strong>Changes since version 4.0: changed "All Contacts" to "All contacts"</strong></li>
```

Comment:
```suggestion
	<li><strong>Changes since version 4.0:</strong> changed "All Contacts" to "All contacts" and no change for the French equivalent static text</li>
```